styles/lib/_modal.css

```diff
@@ -0,0 +1,55 @@
+.modal{
+    display: none;
+    position: fixed;
+    z-index: 100;
```

Comment:
Avoid giving such high values to the z-index. You can instead make it 2 or 3.

Reply:
I gave it because I want modal on top of everything while appearing.